Modelica_LinearSystems2/Controller/Templates/SimpleObserverStateSpaceControl.mo

```diff
@@ -9,7 +9,8 @@ partial model SimpleObserverStateSpaceControl
   Modelica.Blocks.Math.Feedback feedback[feedbackMatrix.nout]
     annotation (Placement(transformation(extent={{-30,-10},{-10,10}})));
   replaceable Internal.PlantTemplate plant(n=
-        feedbackMatrix.nin, m=feedbackMatrix.nout) constrainedby Internal.PlantTemplate
+        feedbackMatrix.nin, m=feedbackMatrix.nout) constrainedby
+    Internal.PlantTemplate

```

Comment:
I guess this change brakes a good readability of the code.  Could you revert this? (Ditto in the following files/classes.)

Reply:
That should now be corrected.